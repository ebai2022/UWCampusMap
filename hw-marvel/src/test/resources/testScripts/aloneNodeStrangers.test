# Testing a node that does not share any edges with the other nodes

LoadGraph g strangers.csv

FindPath g Fake Andrew

FindPath g Ethan Fake