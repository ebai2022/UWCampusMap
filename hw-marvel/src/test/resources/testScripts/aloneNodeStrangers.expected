# Testing a node that does not share any edges with the other nodes

loaded graph g

path from Fake to Andrew:
no path found

path from Ethan to Fake:
no path found